app/javascript/components/patient/assessment/actions/LastDateExposure.js

```diff
@@ -196,6 +201,8 @@ class LastDateExposure extends React.Component {
             ariaLabel="Last Date of Exposure Input"
             isClearable
           />
+          { this.props.continuous_exposure_enabled && (
+          <div>
```

Comment:
please run eslint on this file (and all the other ones too)

Reply:
https://github.com/SaraAlert/SaraAlert/blob/985cc43322dd9e8743a0b6e5935220a2b1155f90/app/javascript/components/patient/assessment/actions/LastDateExposure.js#L205